### PR TITLE
Add nightly ci framework

### PR DESCRIPTION
**Pull request by congqixia**
See also #328 

/kind enhancement

Signed-off-by: Congqi Xia <congqi.xia@zilliz.com>

### CODE REVIEW

File: .github/workflows/nightly_ci.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    #         │ │ │ │ │
+    - cron:  '0 18 * * *'
+
+jobs:
+  nightly:
+    name: Run Nightly CI
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.6]
```
Comment (XuanYang-cn) [inline .github/workflows/nightly_ci.yml:23]: why need python？

Comment (congqixia) [reply .github/workflows/nightly_ci.yml:23]: Legacy from pymilvus, removing it

**Comment by congqixia** [comment]
manual add `ci-passed` for workflow change

**Pull request merged**